### PR TITLE
feat: Add showDomainTotal to Column Chart 

### PR DESCRIPTION
**Pull request by GordonSmith**
## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [x] I have raised new issues to address them separately

### CODE REVIEW

**Review by jeclrsg — APPROVED** [review]

**Comment by GordonSmith** [comment]
@jeclrsg can you re-review:
* Enabled Total to work on stacked and regular column charts
* Moved Total text out of the rect rendering loop and into the "row" rendering loop

File: packages/chart/src/Column.ts [hunk 1]
```diff
@@ -209,7 +205,7 @@ export class Column extends XYAxis {
                                 valueText = d3Format(context.showValueAsPercentFormat())(valueText / seriesSum);
                                 break;
                             case "domain":
-                                const domainSum = typeof dm.sum !== "undefined" ? dm.sum : domainSums[d.idx];
+                                const domainSum = typeof dm.sum !== "undefined" ? dm.sum : domainSums[d.idx - 1];
```
Comment (jeclrsg) [inline packages/chart/src/Column.ts:208]: I missed this before, but could valueText end up as NaN when dm.sum is undefined and d.idx is 0 (domainSums[-1])?

**Review by jeclrsg — CHANGES_REQUESTED** [review]
@GordonSmith Just noticed one array index issue (maybe) that I didn't see the first time

File: packages/chart/src/Column.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/chart/src/Column.ts:208]: That is taken care of with the filter here:  https://github.com/hpcc-systems/Visualization/pull/4174/files#diff-d152ca42ea481d31dcf64caff056f9748655185f1cc12aa26ae12d8f685c332cR157

**Review by jeclrsg — APPROVED** [review]

**Pull request merged**